bootstrap/bootStrap.sh

```diff
@@ -463,25 +467,32 @@ echo "Configuring Instance (output: $configLog)"
 echo -n "    Cloning: ${repoLocation} branch: ${repoBranch}  "
 #Clone this repo
 ssh -q -i "${keyFile}" "${instanceUser}@$PublicDnsName" \
-	"(sleep5;set-x;git clone ${repoLocation}\
+	"(sleep 5;set -x;git clone ${repoLocation}\
 	 --branch ${repoBranch} ~/fabric-skeleton) ">> $configLog 2>&1 &
 spinner $! 1
 
 
 #Do fetch and pull
 ssh -q -i "${keyFile}" "${instanceUser}@$PublicDnsName" \
 	"(set -x;cd ~/fabric-skeleton;ls;git fetch --all;git pull)" >> $configLog 2>&1 &
-spinner $! 
+spinner $!
+
+ssh -q -i  "$keyFile" "${instanceUser}@${PublicDnsName}" \
+	"sudo  bash -c ' echo LC_ALL="en_US.UTF-8" >> /etc/default/locale'" >> $configLog 2>&1 &
 
 echo -n "    Installing development software  "
 #install dev software
 ssh -q -i  "$keyFile" "${instanceUser}@$PublicDnsName" \
 	"sudo -u root -H ~/fabric-skeleton/bootstrap/instanceConfig.sh" >> $configLog 2>&1 &
 spinner $! 1
 
+#workaround for upgrading PyYaml
+ssh -q -i  "$keyFile" "${instanceUser}@${PublicDnsName}" \
+	"sudo  bash -c 'rm -rf /usr/lib/python2.7/dist-packages/yaml && rm -rf /usr/lib/python2.7/dist-packages/PyYAML-3.11.egg-info'" >> $configLog 2>&1 &
+
 #run pip install for the package
 ssh -q -i  "$keyFile" "${instanceUser}@${PublicDnsName}" \
-	"sudo -u root -H pip install -r \
+	". .bashrc && sudo -u root -H pip install -r \
```

Comment:
loading bashrc this way looks awful.
can we use "bash -c ..." or "sudo -i ..." instead?

Reply:
Fixed